docs/developer-guide/syntax-reference.rst

```diff
@@ -939,6 +939,35 @@ example of converting a BIGINT into a VARCHAR type:
       WINDOW TUMBLING (SIZE 20 SECONDS)
       GROUP BY page_id;
 
+CASE
+~~~~
+
+ **Synopsis**
+
+ .. code:: sql
+
+     CASE
+        WHEN condition THEN result
+        [ WHEN ... ]
+        [ ELSE result ]
+     END
+
+ Currently, KSQL supports a ``searched`` form of CASE expression. In this form, CASE evaluates
+each boolean ``condition`` in WHEN caluses, from left to right. If a condition is true, then it returns the
+corresponding result. If none of the conditions are true, it returns the result from the ELSE clause.
```

Comment:
Specify what happens when none of the when conditions evaluate to true, and there is no ELSE clause.